src/include/iscsi-client/helps.rb

```diff
@@ -88,9 +88,10 @@ def initialize_iscsi_client_helps(include_target)
           ),
         # table of connected targets
         "server_table"   => _(
-          "List of current sessions. To add a new target, select it and press <b>Add</b>.\n" +
-            "To remove it, press <b>Log Out</b>.\n" +
-            "To change the start-up status, press <b>Toggle</b>.\n"
+          "<p>List of current sessions.<br>
```

Comment:
Please use 80 characters per line and mimic the original formatting. (BTW Rubocop prefers joining string literals with a backslash but that can wait until we apply Rubocop tho this package)